include/tipsy/binary-to-float.h

```diff
@@ -120,5 +113,7 @@ inline unsigned char ThirdByte(float f) noexcept
     return FloatBytes(f).third();
 }
 
+inline float minimumEncodedFloat() noexcept { return FloatBytes(255,255,255).f; }
```

Comment:
so, this is an example where we can't be constexpr until C++20?

Reply:
Approved with unconsequential formatting quibble and an offhand question.

Otherwise, these are great changes -- they nail down the ranges and boundaries of the encoding nicely, _and test them_. 
I've never liked most tests, but these are very good.

This is becoming a really tight implementation.